Frontend/src/Components/Dashboard/DataRecordingControl.js

```diff
@@ -66,139 +66,27 @@ export default function DataRecordingControl(props) {
 
     // ------------------------------------------ Data recording controls --------------------------------------------
 
-    const { isOpen, onOpen, onClose } = useDisclosure();
+    // TODO: consider making isProcessing boolean to control when to display a loading spinner

```

Comment:
Can we address TODOs before merging?

Reply:
@jackwherry

Reply:
Removed from source code and moved to an issue on Jira (pretty low priority for now though): https://badgerloop.atlassian.net/browse/SW1-24?atlOrigin=eyJpIjoiODY2NzkxMThmZWY3NDFhOGE2ODk5YjliMTA2YTM1NTUiLCJwIjoiaiJ9